stories/cli/steps.fmf

```diff
@@ -81,14 +81,48 @@
 /execute:
     story: 'Select or adjust the execute step'
     description:
-        Specification of the testing framework which should
-        execute tests.
+        Specification of the executor which should run tests.
     example:
         - tmt run execute
-        - tmt run execute --how=shell
```

Comment:
I think that it would be good to preserve "shell" example.

Reply:
Why? We want to deprecate the old execute methods. So I'd say the documentation should not recommend them in any way.

Reply:
Ok, since shell is default framework, and it's nicely documented, similar example (of framework) is probably not needed. Here I mixed "framework" and "how" in my head.